src/patternfly/components/Table/examples/Table.md

```diff
@@ -1492,32 +1502,35 @@ Note: To apply padding to `.pf-c-table__expandable-row`, wrap the content in `.p
 {{/table}}
 ```
 
-### Accessibility
+### Hoverable accessibility
 | Attribute | Applied to | Outcome |
 | -- | -- | -- |
 | `tabindex="0"` | `.pf-c-table tbody.pf-m-hoverable` | Inserts the hoverable table element into the tab order of the page so that it is focusable. **Required** |
 
-### Usage
+### Hoverable and selectable usage
```

Comment:
Should this be?
```suggestion
### Hoverable and selected usage
```

Reply:
I feel like it should either be `Hoverable and selectable usage` or `Hovered and selected usage` but the example is currently titled `Hoverable and selectable` so it should probably change in all cases or left the same?

Reply:
Looks like it should probably be "hoverable and selected" since those are what the modifiers are called. I think "hoverable" implies "selectable".